src/components/FileRepoSidebar.js

```diff
@@ -67,78 +85,36 @@ const Heading = styled('div')`
   margin-bottom: 15px;
   font-weight: 500;
 `;
-
-const FileRepoSidebar = ({ projectId, index, style, sqon, effects, ...props }) => (
-  <div
-    css={`
-      ${styles.container} ${style};
-    `}
-  >
-    <Heading>
-      File Action <InfoIcon />
-    </Heading>
+const FileRepoSidebar = ({ state, projectId, index, style, sqon, effects, theme, ...props }) => {
+  gen3Key = state.integrationTokens[GEN3];
```

Comment:
re: above comment.. don't bother with this step. use `state.integrationTokens[GEN3];` below

Reply:
or at the very least, initialize `gen3Key` here to avoid unnecessary mutaton

Reply:
how much ever i tried avoiding global for these, i believe these variable needs to be in global scope because of the way ColumnState component handles state. Moved the declaration though to avoid mutation.